ci/build-docker.sh

```diff
@@ -54,8 +95,31 @@ else
   echo "no sanitizer errors found."
 fi
 
-# if document generation is enabled, run it now.
+# If document generation is enabled, run it now.
 if [ "${GENERATE_DOCS}" = "yes" ]; then
-  cd ../bigtable
-  doxygen doc/Doxyfile
+  (cd /v/bigtable ; doxygen doc/Doxyfile)
+fi
+
+# Collect the output from the Clang static analyzer and provide instructions to
+# the developers on how to do that locally.
+if [ "${SCAN_BUILD:-}" = "yes" ]; then
+  if [ -n "$(ls -1d /tmp/scan-build-* 2>/dev/null)" ]; then
+    cp -r /tmp/scan-build-* /v/scan-build-output
+  fi
+  if [ -r scan-build-output/index.html ]; then
+    cat <<_EOF_;
+${COLOR_RED}
+scan-build detected errors.  Please read the log for details. To
+run scan-build locally and examine the HTML output install and configure Docker,
+then run:"
+
+DISTRO=ubuntu DISTRO_VERSION=17.04 SCAN_BUILD=yes NCPU=8 TRAVIS_OS_NAME=linux CXX=clang++ CC=clang ./ci/build-linux.sh"
+
+The HTML output will be copied into the scan-build-output subdirectory."
```

Comment:
Why the quote at EOL, if you are outputting all text until `_EOF_`?

Reply:
Fixed.